### ISSUE TITLE
It doesn't load in laravel 6

### ISSUE THREAD

**Issue by DerekBuntin** [naifalshaye/nova-random-password-generation#5]
I've tried installing this package but it doesn't show in Laravel 6x.

**Comment by eugenefvdm** [comment]
Yep unfortunately I have to agree with you, it doesn't work in Laravel 6.x. I looked in Chrome Inspect Element but can't see any obvious issues.

**Comment by VVBphp** [comment]
the problem can be solved by replacing `->onlyOnForms()` to `->hideFromIndex()->hideFromDetail()`

**Comment by eugenefvdm** [comment]
Thanks @VVBphp that worked!
